testing/trino-test-jdbc-compatibility-old-driver/bin/run_tests.sh

```diff
@@ -10,7 +10,7 @@ maven_run_tests="${maven} clean test ${MAVEN_TEST:--B} -pl :trino-test-jdbc-comp
 
 current_version=$(${maven} help:evaluate -Dexpression=project.version -q -DforceStdout)
 previous_released_version=$((${current_version%-SNAPSHOT}-1))
-first_tested_version=352
+first_tested_version=368
```

Comment:
can we automate that?

eg
```shell
trino master$ git tag --contain $(git rev-list master --since='18 month ago' | tail -1) | sort | head -1
364
```

(to be verified whether `git rev-list master --since` filters by author-date or commit-date; we must use the latter)

Reply:
I like the idea. Updated.

Reply:
The filter uses commit-date as far as I tested and confirmed https://github.com/git/git/commit/96697781e06e1b67aeee35a8798deb37e0b87ef4